README.md

```diff
@@ -1 +1,203 @@
+
+Clone this repository to your desired folder:
+Use git clone command or download ZIP folder
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com/Arsenethierry/capstone-project-aspire-web
+```
+--->
+### Install
+Install this project with:
+npm
+Example command:
+```sh
+  cd capstone-project-aspire-web
+  npm init -y
+```
+### Usage
+To run the project, execute the following command:
+npm start or live server
+Example command:
+```sh
+  GitHub Pages Server
+```
+### Run tests
+To run tests, run the following command:
+npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx eslint .
+```
+### Deployment
+You can deploy this project using:
+GitHub Pages
+Example:
+```sh
+git@github.com/Arsenethierry/capstone-project-aspire-web
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :busts_in_silhouette: Authors <a name="authors"></a>
+:bust_in_silhouette: **KUBAHO LINNE Heaven**
+- GitHub: [@Arsenethierry](https://github.com/Arsenethierry)
+- Twitter: [@KomezudufasheT](https://twitter.com/KomezudufasheT)
+- LinkedIn: [@arsenethierry](https://www.linkedin.com/in/arsenethierry/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+
+## :telescope: Future Features <a name="future-features"></a><br/>
+- [ ] **Adding BackEnd**<br/>
+- [ ] **Making it stylish**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!<br/>
+Feel free to check the [git@Arsenethierry/capstone-project-aspire-web.git](https://github.com/Arsenethierry/capstone-project-aspire-web/issues).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- SUPPORT -->
+
+## :star:️ Show your support <a name="support"></a>
+If you like this project please follow me on github & twitter and also connect on Linkedin.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+- I would like to thank Microverse for this capstone practice project.
+
+- I would like to thank the figma design [Cindy Shin](https://www.behance.net/adagio07) for the original design of this project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FAQ (optional) -->
+
+## :question: FAQ (OPTIONAL) <a name="faq"></a><br/>
+- **How long can it take to build that project**
+
+  - three days according to personal skills
+
+- **Where to find figma for that project!**
+
+  - Figma can be found [here](https://www.figma.com/file/oVgXhwArTvUimhkHDyAaJZ/HTML%2FCSS-%26-JavaScript-capstone-project---Conference-page-(Copy)?type=design&node-id=68-4)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Arsenethierry/capstone-project-aspire-web) licensed.
+
```

Comment:
Please kindly fix the license link to ensure that it navigate to your project LICENSE file.  🤗🛠💪